test/message_bus/message_inbox/inbox_message_hash.js

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const BN = require('bn.js');
+const web3 = require('../../test_lib/web3.js');
+const Utils = require('../../test_lib/utils.js');
+const { AccountProvider } = require('../../test_lib/utils.js');
+
+const MessageInbox = artifacts.require('MessageInboxDouble');
+
+contract('MessageInbox::inboxMessageHash', (accounts) => {
+  let messageInbox;
+  const setupParams = {};
+  const accountProvider = new AccountProvider(accounts);
+  beforeEach(async () => {
+    messageInbox = await MessageInbox.new();
+    setupParams.intentHash = Utils.getRandomHash();
+    setupParams.nonce = new BN(Utils.getRandomNumber(63)); // need to verify the range
+    setupParams.feeGasPrice = new BN(Utils.getRandomNumber(9999));
+    setupParams.feeGasLimit = new BN(Utils.getRandomNumber(10000));
+    setupParams.sender = messageInbox.address;
+    setupParams.messageOutbox = await accountProvider.get();
+    setupParams.metachainId = Utils.generateRandomMetachainId();
+    setupParams.outboxStorageIndex = new BN(Utils.getRandomNumber(63));
+    setupParams.stateRootI = await accountProvider.get();
+    setupParams.maxStorageRootItems = new BN(Utils.getRandomNumber(500));
+  });
+
+  contract('Positive Tests', async () => {
+    it('Should be able to set parameters', async () => {
```

Comment:
Update the it statement as per method it's calling.
It statement is incorrect

Reply:
Updated the statement.